### PR TITLE
Set auth token for all si platform requests

### PR DESCRIPTION
**Pull request by patuwwy**
`si auth set-token <token>` to set platform token

### CODE REVIEW

**Review by alicja-gruzdz — APPROVED** [review]

File: packages/cli/src/lib/commands/auth.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    * Command `si host version`
+    * Get STH version number from package.json file
+    * Log: version number
+    */
+    authCmd
+        .command("set-token")
+        .argument("<token>")
+        .description("Set Platform auth token")
+        .action(async (token: string) => {
+            setConfigValue("token", token);
```
Comment (ErykSol) [inline packages/cli/src/lib/commands/auth.ts:23]: shouldn't we add simple format validation for JWT before save to config?

**Review by ErykSol — COMMENTED** [review]
nice and clean :)

File: packages/cli/src/lib/commands/auth.ts (hunk 1 above)
Comment (patuwwy) [reply packages/cli/src/lib/commands/auth.ts:23]: we can add simple API endpoint to platfrom to validate token or just check here if lenght is correct

Comment (ErykSol) [reply packages/cli/src/lib/commands/auth.ts:23]: I would go for simple format check- lenght and number of dots in string

**Review by ErykSol — APPROVED** [review]

**Pull request merged**